translate/automl/translate_v3_batch_translate_text_with_model.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START translate_v3_batch_translate_text_with_model]
+from google.cloud import translate
+
+
+def batch_translate_text_with_model(
+    input_uri="gs://YOUR_BUCKET_ID/path/to/your/file.type",
```

Comment:
Change `.type` to whatever is supported by the API. 

Probably worth adding a link below to this page for more info about the supported file types. 
https://cloud.google.com/translate/docs/reference/rpc/google.cloud.translation.v3#google.cloud.translation.v3.InputConfig

Reply:
this link is a bit long for including the sample, I have added this link instead 
https://cloud.google.com/translate/docs/supported-formats